pkg/hubble/parser/options/options.go

```diff
@@ -60,6 +61,18 @@ func Redact(logger logrus.FieldLogger, httpQuery, httpUserInfo, kafkaApiKey bool
 	}
 }
 
+// EnableL3L4 configures the Network Policy enrichment.
+func EnableL3L4(logger logrus.FieldLogger, enableL3L4 bool) Option {
+	return func(opt *Options) {
+		opt.EnableL3L4PolicyEnrichment = enableL3L4
+		if logger != nil {
```

Comment:
fyi they're about to merge a change moving to a new logger: https://github.com/cilium/cilium/pull/37921#pullrequestreview-2656776407